src/components/sbb-tab-group/sbb-tab-group.stories.js

```diff
@@ -0,0 +1,404 @@
+};
+
+const iconSlot = {
+  control: {
+    type: 'select',
+  },
+  options: ['app-icon-small', 'train-small', 'swisspass-small', 'pie-small'],
+  table: {
+    category: 'Tab1',
+  },
+};
+
+const amountSlot = {
+  control: {
+    type: 'number',
+  },
+  table: {
+    category: 'Tab1',
+  },
+};
+
+const disabledArg = {
+  control: {
+    type: 'boolean',
+  },
+  table: {
+    category: 'Disabled Tab',
+  },
+};
+
+const basicArgTypes = {
+  label,
+  iconSlot,
+  amountSlot,
+  disabled: disabledArg,
+};
+
+const basicArgs = {
+  label: 'Tab label one',
+  iconSlot: '',
+  amountSlot: '',
+  disabled: true,
+};
+
+export const defaultTabs = DefaultTemplate.bind({});
+export const numbers = NumbersTemplate.bind({});
+export const icons = IconsTemplate.bind({});
+export const numbersAndIcons = IconsAndNumbersTemplate.bind({});
+export const nestedTabGroups = NestedTemplate.bind({});
+export const tintedBackground = IconsAndNumbersTemplate.bind({});
+
+defaultTabs.argTypes = basicArgTypes;
+numbers.argTypes = basicArgTypes;
+icons.argTypes = basicArgTypes;
+numbersAndIcons.argTypes = basicArgTypes;
+nestedTabGroups.argTypes = basicArgTypes;
+tintedBackground.argTypes = basicArgTypes;
+
+defaultTabs.args = JSON.parse(JSON.stringify(basicArgs));
+numbers.args = JSON.parse(JSON.stringify(basicArgs));
+icons.args = JSON.parse(JSON.stringify(basicArgs));
+numbersAndIcons.args = JSON.parse(JSON.stringify(basicArgs));
+nestedTabGroups.args = JSON.parse(JSON.stringify(basicArgs));
+tintedBackground.args = JSON.parse(JSON.stringify(basicArgs));
+
+tintedBackground.decorators = [(Story) => <Story style={'background: #f6f6f6; padding: 2rem;'} />];
+
+/* VARIANTS */
+numbers.args.amountSlot = '77';
+icons.args.iconSlot = iconSlot.options[0];
+numbersAndIcons.args.amountSlot = '16';
+numbersAndIcons.args.iconSlot = iconSlot.options[0];
+nestedTabGroups.args.amountSlot = '16';
+nestedTabGroups.args.iconSlot = iconSlot.options[0];
+tintedBackground.args.amountSlot = '16';
+tintedBackground.args.iconSlot = iconSlot.options[0];
+
+defaultTabs.documentation = {
+  title: 'Default Tabs',
+};
+numbers.documentation = {
+  title: 'Tabs with numbers',
+};
+icons.documentation = {
+  title: 'Tabs with icons',
+};
+numbersAndIcons.documentation = {
+  title: 'Tabs with numbers and icons',
+};
+tintedBackground.documentation = {
+  title: 'Tab group on non-white background',
+};
+
+defaultTabs.decorators = [
+  (Story) => (
+    <div style={'padding: 2rem'}>
+      <Story />
+    </div>
+  ),
+];
+
+numbers.decorators = [
+  (Story) => (
+    <div style={'padding: 2rem'}>
+      <Story />
+    </div>
+  ),
+];
+
+icons.decorators = [
+  (Story) => (
+    <div style={'padding: 2rem'}>
+      <Story />
+    </div>
+  ),
+];
+
+numbersAndIcons.decorators = [
+  (Story) => (
+    <div style={'padding: 2rem'}>
+      <Story />
+    </div>
+  ),
+];
+
+nestedTabGroups.decorators = [
+  (Story) => (
+    <div style={'padding: 2rem'}>
+      <Story />
+    </div>
+  ),
+];
```

Comment:
you can store the template resp. the decorator into a const. and then reuse it. maybe it's also possible to do it like

```
nestedTabGroups.decorators = numbersAndIcons.decorators = [
  (Story) => (
    <div style={'padding: 2rem'}>
      <Story />
    </div>
  ),
];
```